### PR TITLE
changes implementation of flipScheduleEnabled and flipExecutionEnable…

### PR DESCRIPTION
**Pull request by miguelantonio**
solving issue 1468

### CODE REVIEW

**Comment by gschueler** [comment]
@miguelantonio can you add a unit test to verify the fix?

**Comment by miguelantonio** [comment]
@gschueler test added!

**Comment by miguelantonio** [comment]
@gschueler 1ee4bce added fix to issues #1485 and #1475 (restart enabled schedule toggle)

We test this first using the development branch and it reproduced, it was because at startup ir was using the incorrect funciton, now it uses the correct one, we tested with the fix and it doesn't schedule disabled scheduled jobs

File: test/api/test-job-flip-scheduleEnabled-scheduler-bug.sh [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    runurl="${APIURL}/jobs/export?project=${projname}&format=xml"
+    params=""
+
+    # get listing
+    docurl ${runurl}?${params} > $DIR/curl.out || fail "failed request: ${runurl}"
+
+    assert 1 $(xmlsel "count(/joblist/job/schedule/month)" $DIR/curl.out) "can't find tag job/schedule/month"
+    assert 1 $(xmlsel "count(/joblist/job/schedule/time)" $DIR/curl.out) "can't find tag job/schedule/time"
+    assert 1 $(xmlsel "count(/joblist/job/schedule/year)" $DIR/curl.out) "can't find tag job/schedule/year"
+    assert 1 $(xmlsel "count(/joblist/job/schedule/weekday)" $DIR/curl.out) "can't find tag job/schedule/weekday"
```
Comment (gschueler) [inline test/api/test-job-flip-scheduleEnabled-scheduler-bug.sh]: need to verify the actual value of `weekday`

**Comment by gschueler** [comment]
if i run the `test/api/test-job-flip-scheduleEnabled-scheduler-bug.sh` test script without applying any of these changes, it still succeeds, so it doesn't seem to verify the fix

**Comment by miguelantonio** [comment]
Greg, we were testing using the API but the bug doesn't happen in that channel, it only does by using the Web, we are right now building the right test and checking days

basically we are now trying to execute a GET simulating the use of the web http://localhost:4440/project/<project>/job/flipScheduleEnabled/<job>?scheduleEnabled=true/false

will commit as soon as it reproduce with the test

Also, do you have any hint on logon as user with curl? (headers?)

**Comment by gschueler** [comment]
@miguelantonio I see. In that case, don't worry about the api-level test. If you want I can just add a unit test to verify the correct service layer method is called for that action

**Comment by miguelantonio** [comment]
oh, thank you, that's great

**Comment by gschueler** [comment]
ok, here's a trial: I committed a broken test.  merging this PR should fix it :)

**Comment by miguelantonio** [comment]
alright :)

**Pull request merged**